src/Query/ReadObject.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+namespace Softonic\GraphQL\Query;
+
+interface ReadObject
+{
```

Comment:
Should the ReadObjects implement some method?

Reply:
No, it has just magic methods and is used to "group" both QueryItem and CollectionItem.